apps/source_academy/priv/repo/migrations/20170626081409_create_users.exs

```diff
@@ -0,0 +1,15 @@
+defmodule SourceAcademy.Repo.Migrations.CreateUsers do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :name, :string
+      add :email, :string
+      add :is_admin, :boolean, default: false
+
+      timestamps()
+    end
+
+    create index(:users, [:email], unique: true)
```

Comment:
In addition to this, try to create partial index on role=student since listing all student names is actually quite frequent

See [Partial Index](http://rny.io/elixir/ecto/postgresql/2015/08/31/postgresql-indexing-with-ecto.html)